### PR TITLE
fixes bug 1272050 - Report index breaks on failing counting correlations

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ var SignatureCorrelations = (function() {
     var container = $('#mainbody');
     return {
         showSignatureCorrelationsTab: function() {
+            if (!container.data('product')) {
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js]: You're verifying that ``product`` is false, but in the comment you talk about ``platform``. Which one holds the truth?

File: webapp-django/crashstats/crashstats/views.py [hunk 2]
```diff
@@ -69,6 +69,10 @@
 
 
 def make_correlations_count_cache_key(product, version, platform, signature):
+    assert product
+    assert version
+    assert platform
+    assert signature
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/views.py]: I don't really like using ``assert`` to much in the code. You already make sure all of those have a value before calling the function, what's the point of verifying that twice?

Comment (peterbe) [inline webapp-django/crashstats/crashstats/views.py]: I used it to debug. I took care of the real problem elsewhere later but forgot to delete these lines.

**Comment by peterbe** [comment]
@adngdb Nits addressed.

**Pull request merged**